### ISSUE TITLE
Clarification on pretrained NMTs

### ISSUE THREAD

**Issue by aaaallleen** [bert-nmt/bert-nmt#82]
Hi, I am currently researching low-resource languages and would like to know if fusing BERT would increase the performance. There are some questions I would like to know.

In your readme, the steps instructed us to obtain a vanilla NMT using fairseq. I am currently training using Marian as a baseline, is it possible to use a Marian pretrained NMT as the warmup-NMT file? 

I noticed that a .pt checkpoint model should be loaded and I transformed my Marian model to a pytorch model using a script provided by the OpusMT team. Could this model be utilized with the BERT-NMT structure? Thank you.
